roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -141,7 +143,9 @@
     dest: "{{ tempdir }}/templates/oauth-client.yml"
   vars:
     kibana_hostname: "{{ openshift_logging_kibana_hostname }}"
+    kibana_ops_hostname: "{{ openshift_logging_kibana_ops_hostname }}"
     secret: "{{ oauth_secret }}"
+  when: do_oauth_client is defined and do_oauth_client | bool
```

Comment:
ideally we would let the oauth creation be idempotent. If we can ensure that we are reusing the same oauth_secret and session_secret we shouldn't need to do this